starport/interface/cli/starport/cmd/network_chain_join.go

```diff
@@ -34,14 +43,15 @@ func NewNetworkChainJoin() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 	}
 	c.Flags().AddFlagSet(flagSetHomes())
+	c.Flags().String(flagGentx, "", fmt.Sprintf("Path to a gentx file (optional, requires --%s to be provided)", flagPeer))
```

Comment:
Seems like both flags can be provided separately now, right? If it's the case we should remove the statement `(optional, requires --%s to be provided)`, below as well

Reply:
solved by cb79727.